scapy/config.py

```diff
@@ -329,6 +330,22 @@ def __set__(self, obj, val):
         obj._logLevel = val
 
 
+def _version_checker(module, minver):
```

Comment:
Why do you need this in this PR?

Reply:
Importing distutils is creating a DeprecationWarning (imports imp). That avoids importing it, by copying the LooseVersion behavior

Reply:
Could you add some specific unit tests for this function?